systemd/manager.go

```diff
@@ -106,6 +106,14 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if len(u.Contents["Install"]) > 0 {
```

Comment:
It doesn't matter. Current fleet behavior - skip empty sections. If we will keep these sections - systemd will still process such units.